src/main/scala/org/codeoverflow/chatoverflow/requirement/parameter/UriParameterImpl.scala

```diff
@@ -0,0 +1,20 @@
+package org.codeoverflow.chatoverflow.requirement.parameter
+
+import org.codeoverflow.chatoverflow.api.io.parameter.UriParameter
+import org.codeoverflow.chatoverflow.registry.Impl
+import java.net.URI
+
+@Impl(impl = classOf[UriParameter])
+class UriParameterImpl extends UriParameter {
+  private var value: URI = null
+
+  override def getType: Class[URI] = classOf[URI]
+
+  override def serialize(): String = get().toString
+
+  override def get(): URI = value
+
+  override def deserialize(value: String): Unit = set(new URI(value))
```

Comment:
Please add exception handling here for `URISyntaxException` that may be thrwon by the `URI` constructor.

Reply:
I think thorwing this exception uncought is intended behaviour here. 
Any occuring exceptions are cought by the ConfigurationService and will inform the framework that the Parameter couldn't be set.

Reply:
After looking into it, throwing exceptions is the correct behavior, the rest api also reports to the gui that it has failed to set the requirement if a exception is thrown. 

This means that all error handling of all other parameters should be removed.